examples/always_returning_example.py

```diff
@@ -0,0 +1,21 @@
+def loop():
+    for j in range(0, 1):
```

Comment:
Oh!!! Sorry I did not notice that mistake. Maybe we could do a checker for that(Especially beneficial for careless students like me.)